crates/node-maintainer/src/lockfile.rs

```diff
@@ -31,6 +47,26 @@ impl Lockfile {
         &self.packages
     }
 
+    pub fn to_json(&self) -> JsonDocument {
+        let mut doc = json!({});
+        doc["name"] = json!(self.root.name.as_ref());
+        doc["version"] = json!(self.root.version);
+        doc["lockfileVersion"] = json!(self.version);
+        doc["requires"] = json!(true);
+        let mut packages = self.packages.iter().collect::<Vec<_>>();
+        packages.sort_by(|(a, _), (b, _)| a.cmp(b));
+        let mut deps = IndexMap::new();
+        let mut deps_v1 = IndexMap::new();
+        deps.insert("".to_string(), self.root.to_json());
+        for (name, pkg) in packages {
+            deps.insert("node_modules/".to_string() + name.as_str(), pkg.to_json());
```

Comment:
This actually needs to be the path of the dependency on disk. Nested dependencies will look like `node_modules/foo/node_modules/bar`.